Guide_1_Send_Payment.md

```diff
@@ -37,11 +117,12 @@ The key parameters that you need to provide are:
 
 #### Debtor and Beneficiary Data
 
-To identify the sending and receiving parties of the payment, you need to provide an `account_name`, the `account_number`, the `account_number_code`, as well as the `bank_id` and `bank_id_code` for the bank the account is registered with.
+To identify the sending and receiving parties of the payment, you need to provide an `account_name`, the `account_number`, the `account_number_code`, as well as the `bank_id` and `bank_id_code` for the bank the account is registered with. 
+
+In this example, use your UK sortcode for the sending party's `bank_id`. 
+
+The `bank_id_code ` attribute denotes the type of the `bank_id`. Since only domestic UK accounts are used in this example, the `bank_id_code` is preset as `GBDSC`.
 
-```json
-Code for payment resource creation here!
-```
 Upon success, the call returns with a payment ID that you'll need below to send the payment.
```

Comment:
Actually you already specify the payment ID before, so you dont need this here.

Reply:
True, good point! I'll remove that.